_posts/2025-08-24-update.md

```diff
@@ -21,19 +21,9 @@ slug: 2025-08-24-update
 
 ## Other Merges
 
-*
-
-## Promotions
-
-*
-
-## Deprecated
+*  [Storage counting](https://github.com/kubernetes/kubernetes/pull/133604) all objects instead of objects for resource is fixed
+*  Data race around [claimsToAllocate](https://github.com/kubernetes/kubernetes/pull/133587) is fixed
```

Comment:
These do not belong under Deprecated.  They belong under Other Merges.  The Deprecated section is for deprecations only.

Also, some rewordings:

```suggestion
*  [Count storage types accurately](https://github.com/kubernetes/kubernetes/pull/133604) when filtering per type
*  Prevent data race around [claimsToAllocate](https://github.com/kubernetes/kubernetes/pull/133587) 
```